apiserver/facades/agent/uniter/goal-state_test.go

```diff
@@ -27,68 +25,24 @@ import (
 // versions. It's not intended to be used directly or registered as a
 // suite, but embedded.
 type uniterGoalStateSuite struct {
-	testing.JujuConnSuite
-
-	authorizer apiservertesting.FakeAuthorizer
-	resources  *common.Resources
-	uniter     *uniter.UniterAPI
-
-	machine0      *state.Machine
-	machine1      *state.Machine
-	machine2      *state.Machine
-	logging       *state.Application
-	wordpress     *state.Application
-	wpCharm       *state.Charm
-	mysql         *state.Application
-	wordpressUnit *state.Unit
-	mysqlUnit     *state.Unit
+	uniterSuiteBase
+
+	machine2 *state.Machine
```

Comment:
it seems odd that we would strip out machine 0 and 1 but not 2. did we miss a step?

Reply:
It has `machine0` and `machine1` via the `uniterSuiteBase`. I just removed some repetition by using the base as intended.